packages/instant/src/util/error.ts

```diff
@@ -60,6 +60,9 @@ export const errorUtil = {
         if (error instanceof Error) {
             bestMessage = humanReadableMessageForError(error, asset);
         }
+        if (_.isString(error)) {
```

Comment:
I personally like the pattern of raising an error with a constant, and then turning into a human readable version here. This way we have one place to look at all the human readable errors we produce.  See https://github.com/0xProject/0x-monorepo/pull/1193/files for an example of this.